test/karma/mocks/test-mocks.js

```diff
@@ -0,0 +1,6 @@
+Float64Array = function (num) {return num};
+sjcl = {};
+sjcl.misc = {}
+Raven = {};
+Raven.config = function () {return{'install': function(){}}};
+Options = {'SENTRY_DSN': ''};
```

Comment:
just a bunch of quick global mocks to get the tests to run... to be improved upon